packages/lwc-language-server/src/__tests__/lwc-server.test.ts

```diff
@@ -76,26 +81,44 @@ describe('handlers', () => {
     describe('#onCompletion', () => {
         it('should return a list of available completion items in a javascript file', async () => {
             const params: CompletionParams = {
-                textDocument: { uri },
+                textDocument: { uri: jsUri },
                 position: {
                     line: 0,
                     character: 0,
                 },
                 context: {
-                    triggerCharacter: '{',
+                    triggerCharacter: '.',
                     triggerKind: 2,
                 },
             };
 
             await server.onInitialize(initializeParams);
             const completions = await server.onCompletion(params);
             const labels = completions.items.map(item => item.label);
-            expect(labels).toBeArrayOfSize(19);
-            expect(labels).toInclude('handleToggleAll');
-            expect(labels).toInclude('handleClearCompleted');
+            expect(labels).toBeArrayOfSize(8);
+            expect(labels).toInclude('c/todo_util');
+            expect(labels).toInclude('c/todo_item');
+        });
+
+        it('should not return a list of completion items in a javascript file for open curly brace', async () => {
+            const params: CompletionParams = {
+                textDocument: { uri: jsUri },
+                position: {
+                    line: 0,
+                    character: 0,
+                },
+                context: {
+                    triggerCharacter: '{',
+                    triggerKind: 2,
```

Comment:
total nit: do we have an enum for triggerKind?

Reply:
So this comes from the vscode-language-server-protocol, where it looks like CompletionTriggerKind is declared as a type. Not sure if we can/want to import that here.

Reply:
ah gotcha..yeah looks like we can just import and use that type. 
<img width="661" alt="Screen Shot 2022-03-15 at 4 13 09 PM" src="https://user-images.githubusercontent.com/76090802/158473709-360647e7-854a-4251-a466-b9323c5b223d.png">

Reply:
Cool beans, just added!